File: frontend/src/app/enum/cancel-reason.ts
export enum CancelReason {
  NO_INTEREST = 'NO_INTEREST',
  LOCATION_REMOVED = 'LOCATION_REMOVED',
  SCAM = 'SCAM',
}

export const reasonToDisplayText = {
  NO_INTEREST: 'Kein Interesse',
  LOCATION_REMOVED: 'Location entfernt',
  SCAM: 'Scam',
};

File: backend/src/main/java/at/ac/tuwien/sepm/groupphase/backend/enums/CancelReason.java
package at.ac.tuwien.sepm.groupphase.backend.enums;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public enum CancelReason {

    NO_INTEREST("Kein Interesse"),
    LOCATION_REMOVED("Location entfernt"),
    USER_REMOVED("Anderer User gelöscht"),
    SCAM("Scam");

    private final String displayValue;
}
